### PR TITLE
GSoC-2022_CLUE-kd proposal

### PR DESCRIPTION
**Pull request by tonydp03**
CLUE-kd proposal for GSoC 2022.

@felicepantaleo @waredjeb @rovere

### CODE REVIEW

File: gsoc/2022/mentors.md [hunk 1]
```diff
@@ -27,3 +27,7 @@ layout: plain
 * Mayank Sharma[Mayank Sharma](mailto:mayank.sharma@cern.ch) CERN
 * Mark Smith [mark.smith1@imperial.ac.uk](mailto:mark.smith1@imperial.ac.uk) imperialcollege
 * Anna Zaborowska[Anna Zaborowska](mailto:anna.zaborowska@cern.ch) CERN
+* Felice Pantaleo[Felice Pantaleo](mailto:felice.pantaleo@cern.ch) CERN
```
Comment (hegner) [inline gsoc/2022/mentors.md:30]: please insert mentors in alphabetical order

File: _gsocproposals/2022/proposal_CLUE-kD-python-library.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Desirable Skills
+
+ * Experience with Python library development
+ * Experience with Python C++ bindings
+ * Experience with C++17
+
+## Mentors
+
+ * [Felice Pantaleo](mailto:felice.pantaleo@cern.ch) (CERN)
```
Comment (hegner) [inline _gsocproposals/2022/proposal_CLUE-kD-python-library.md:46]: could you specify/highlight the main mentor?

File: _gsocproposals/2022/proposal_CLUE-kD-python-library.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ * [Tony Di Pilato](mailto:tony.dipilato@cern.ch) (CASUS HZDR)
+ * [Wahid Redjeb](mailto:wahid.redjeb@cern.ch) (CERN RWTH)
```
Comment (hegner) [inline _gsocproposals/2022/proposal_CLUE-kD-python-library.md:48]: RWTH and CASUS HZDR I do not see as organizations. Can you add?

**Review by hegner — CHANGES_REQUESTED** [review]

File: _gsocproposals/2022/proposal_CLUE-kD-python-library.md (hunk 3 above)
Comment (tonydp03) [reply _gsocproposals/2022/proposal_CLUE-kD-python-library.md:48]: Added organizations

File: _gsocproposals/2022/proposal_CLUE-kD-python-library.md (hunk 2 above)
Comment (tonydp03) [reply _gsocproposals/2022/proposal_CLUE-kD-python-library.md]: Highlighted the main mentor.

File: gsoc/2022/mentors.md (hunk 1 above)
Comment (tonydp03) [reply gsoc/2022/mentors.md]: Done.

**Review by hegner — APPROVED** [review]

**Pull request merged**